### ISSUE TITLE
FutureWarning: is_categorical_dtype is deprecated and will be removed in a future version.

### ISSUE THREAD

**Issue by newini** [scikit-learn-contrib/category_encoders#430]
## Expected Behavior
This is the future warning so should be fixed.

## Actual Behavior
```
/var/lang/lib/python3.11/site-packages/category_encoders/utils.py:50: FutureWarning: is_categorical_dtype is deprecated and will be removed in a future version. Use isinstance(dtype, CategoricalDtype) instead
  return pd.api.types.is_categorical_dtype(dtype)
```

- See Pandas GitHub : https://github.com/pandas-dev/pandas/pull/52527

## Steps to Reproduce the Problem
This is the future warning so should be fixed.

## Specifications
  - Version: '2.6.2'
  - Platform: Python 3.11.4
  - Subsystem: WSL2
  - Pandas version: '2.1.2'

**Comment by newini** [comment]
I found that the warning is fixed in master branch. https://github.com/scikit-learn-contrib/category_encoders/blob/master/category_encoders/utils.py#L53C2-L53C2

Why not release?

**Comment by PaulWestenthanner** [comment]
I just released 2.6.3. Sorry for being a little late with this.
